styles/index.less

```
@@ -0,0 +1,60 @@
+@import "./common-styles/index";
+
+// TODO: asciinema is included in every build
+
+body {
+  & > header {
+    &:extend(.navigation-bar all);
+
+    margin-bottom: 0;
+  }
+
+  // TODO: why is this not possible?
+  //& > div.page-title {
+  //   &:extend(.page-title all);
+  //}
+
+  & > div.page-description {
+      //TODO: &:extend(.page-description all);
+      #container();
+
+      font-weight: bold;
+      font-size: 120%;
+      color: #theme.color[gray-dark];
+  }
+
+  & > main > #alerts {
+    #container();
+
+    & > .notice-box {
+      & > .issue-age {
+        display: inline-block;
+        min-width: 6em;
+      }
+      & > a.issue-link {
+        color: #theme.color[blue-dark];
+      }
+    }
+  }
+
+  & > main > div#content {
+    #container();
+
+    & > table.styled-table  { 
```

```suggestion
    table.styled-table  { 
```

No need to get carried away with `>` too much. It's already quite precise with `& > main > div#content`. And using too many direct descendent selectors breaks when the DOM is reworked.

By following [rscss](https://rscss.io/), we can be pretty confident we're not accidentally selecting too much.